notices/feed.xml

```diff
@@ -7,8 +7,8 @@ layout:
   <channel>
     <title>{{ site.title | xml_escape }} - Notices</title>
     <description>Notices are our means to communicate and document changes in the project to contributors, core developers, users, and the community.</description>
-    <link>{% link notices/notices.md %}</link>
-    <atom:link href="{% link notices/feed.xml %}" rel="self" type="application/rss+xml"/>
+    <link>{{ "/notices" | absolute_url }}</link>
```

Comment:
I'm seeing relative URLs on the preview feed below despite this `absolute_url` filter. Not sure if this is some kind of Netlify side-effect

https://deploy-preview-115--docs-rapids-ai.netlify.app/notices/feed.xml

Reply:
Yes this is a netlify thing where we unset the `site.url` so `absolute_url` has nothing to add. It works in production and previews though

Reply:
https://docs.rapids.ai/notices/feed.xml < here from #112 you can see `link` and `guid` tags are populated with `absolute_url` and work. The current `link` liquid function doesn't resolve